composer.json

```diff
@@ -99,7 +98,8 @@
         "sonata-project/seo-bundle": "^2.1",
         "symfony/monolog-bundle": "If you want to log exceptions produced when dealing with images.",
         "tilleuls/ckeditor-sonata-media-bundle": "^1.0",
-        "twig/extra-bundle": "Auto configures the Twig Intl extension"
+        "twig/extra-bundle": "Auto configures the Twig Intl extension",
+        "sonata-project/formatter-bundle": "^3.4 || ^4.0"
```

Comment:
Please add a textual description, no version information.

Reply:
I left that in by mistake. I think it can be removed altogether. MediaBundle just provides some services to the Formatter Bundle, not the other way around.